packages/react-composites/src/composites/CallWithChatComposite/state/CallWithChatAdapterState.ts

```diff
@@ -40,11 +40,11 @@ export interface CallWithChatAdapterUiState {
   /**
    * Files being uploaded by a user in the current thread.
    * Should be set to null once the upload is complete.
-   * Array of type {@link AttachmentUploadsUiState}
+   * Array of type {@link _AttachmentUploadsUiState}
    *
    * @beta
    */
-  attachmentUploads?: AttachmentUploadsUiState;
+  attachmentUploads?: _AttachmentUploadsUiState;
```

Comment:
What needs to change here- visibility or naming?

Reply:
visibility, updated thanks